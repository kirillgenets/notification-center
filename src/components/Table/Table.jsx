import React from 'react';
import TableHeading from '../TableHeading/TableHeading';
import TableBody from '../Notificatons/Notifications';
import styles from './Table.css';

const Table = () => {
  return (
    <table className={styles.table}>
      <TableHeading />
      <TableBody />
    </table>
  )
}

export default Table;