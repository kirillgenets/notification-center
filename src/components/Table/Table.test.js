import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Table from './Table';

configure({
  adapter: new Adapter(),
});

it('Table renders correctly', () => {
  const tree = shallow(
    <Table />,
  );

  expect(tree).toMatchSnapshot();
});
