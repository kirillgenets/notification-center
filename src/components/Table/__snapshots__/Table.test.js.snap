// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table renders correctly 1`] = `
<table>
  <TableHeading />
  <Connect(Notifications) />
</table>
`;
